hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/ContractCallSuccessfulResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.hedera.mirror.web3.controller;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import lombok.Data;
+import javax.validation.constraints.NotNull;
+
+@Data
+abstract class ContractCallSuccessfulResponse implements ContractCallResponse {
```

Comment:
Not sure why it's abstract and the interface serves no purpose with the knowledge that the error response is generic to all APIs.

```suggestion
@Data
class ContractCallResponse {
```